resources/DirectoryLister.php

```diff
@@ -61,7 +61,35 @@ public function __construct() {
         $this->_themeName = $this->_config['theme_name'];
 
     }
-
+	/**
+	* If it is has password enabled
+	* 
+	* @return true if enabled, false if not
+	* @access public
+	*/
+	public function isPasswordProtected () {
```

Comment:
I don't see this function actually used anywhere.  Did you forget to push some of your changes?